openshift/buildConfig-template.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: naming-service-buildconfig
+  thoth: 0.1.0
+metadata:
+  name: naming-service-buildconfig
+  annotations:
+    description: This is Thoth Naming Service BuildConfig, this template is meant to be used by Jenkins, but could also be used by humans...
+    openshift.io/display-name: 'Thoth Core: User API BuildConfig'
+    version: 0.2.0
+    tags: poc,thoth,thoth-naming-service,ai-stacks
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: This is Thoth Naming Service BuildConfig, this template is meant to be used by Jenkins, but could also be used by humans...
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+
+objects:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: naming-service
+    labels:
+      app: thoth-core
+      component: naming-service
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: "naming-service:${IMAGE_STREAM_TAG}"
+    source:
+      type: Git
+      git:
+        uri: ${GITHUB_URL}
+        ref: ${GITHUB_REF}
+    strategy:
+      type: Source
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: python-36-centos7:latest
+        env:
+          - name: ENABLE_PIPENV
+            value: '1'
+          - name: UPGRADE_PIP_TO_LATEST
+            value: ''
+          - name: WEB_CONCURRENCY
+            value: '4'
+    triggers:
+    - type: ImageChange
+      imageChange: {}
+
+parameters:
+- description: Name of the github repository for Thoth's User API
+  displayName: Git Repository
+  required: true
+  name: GITHUB_URL
+  value: 'https://github.com/thoth-station/naming-service'
+
+- description: Git reference to be used for Thoth's User API
+  displayName: Git Reference
+  required: true
+  name: GITHUB_REF
+  value: 'master'
+
+- description: Tag of the output ImageStream the resulting container image should go to
+  displayName: ImageStream Tag
+  required: true
+  name: IMAGE_STREAM_TAG
+  value: 'latest'
```

Comment:
Some IDEs/text editors are just broken :smiley:

Reply:
:/

Reply:
it's Visual Studio Code, from our friends at Microsoft!